### PR TITLE
update generate and bootstrap to use generated clients

### PR DESCRIPTION
**Pull request by deads2k**
replaces https://github.com/openshift/origin/pull/16550

### CODE REVIEW

File: pkg/oc/admin/project/new_project.go [hunk 1]
```diff
@@ -55,18 +54,10 @@ func NewCmdNewProject(name, fullName string, f *clientcmd.Factory, out io.Writer
 		Short: "Create a new project",
 		Long:  newProjectLong,
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := options.complete(args); err != nil {
+			if err := options.complete(f, args); err != nil {
 				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
-			var err error
```
Comment (deads2k) [inline pkg/oc/admin/project/new_project.go]: @enj as promised.

File: pkg/oc/bootstrap/docker/up.go [hunk 2]
```diff
@@ -1218,16 +1220,17 @@ func (c *ClientStartConfig) Factory() (*clientcmd.Factory, error) {
 }
 
 // Clients returns clients for OpenShift and Kube
-func (c *ClientStartConfig) Clients() (*client.Client, kclientset.Interface, error) {
+// FIXME: Refactor this to KubernetesInternal() call.
+func (c *ClientStartConfig) Clients() (interface{}, kclientset.Interface, error) {
```
Comment (deads2k) [inline pkg/oc/bootstrap/docker/up.go]: wow, this was ugly.  I'll revisit it.

**Comment by deads2k** [comment]
This is @mfojtik's pull. (my commit was already lgtm'ed).  Tagging.

lgtm

**Comment by enj** [comment]
> as promised

Thanks.

/lgtm

**Comment by deads2k** [comment]
> pkg/oc/admin/project/new_project.go
> pkg/oc/bootstrap/docker/openshift/logging.go
> test/integration/authorization_test.go
> test/integration/build_admission_test.go
> test/integration/clusterquota_test.go
> test/integration/clusterresourceoverride_admission_test.go
> test/integration/deploy_scale_test.go
> test/integration/deploy_trigger_test.go
> test/integration/dockerregistry_pullthrough_test.go
> test/integration/endpoint_admission_test.go
> test/integration/extensions_api_deletion_test.go
> test/integration/extensions_api_disabled_test.go
> test/integration/front_proxy_test.go
> test/integration/gc_default_test.go
> test/integration/imagechange_buildtrigger_test.go
> test/integration/login_test.go
> test/integration/oauth_serviceaccount_client_test.go
> test/integration/ownerrefs_test.go
> test/integration/policy_commands_test.go
> test/integration/project_test.go
> test/integration/restrictusers_test.go
> test/integration/scc_test.go
> test/integration/scopes_test.go
> test/integration/sdn_test.go
> test/integration/service_account_test.go

@soltysh if I ever volunteer to rebase @mfojtik's pulls again, remind me of this.

**Comment by deads2k** [comment]
@mfojtik the original didn't compile :(

**Pull request merged**